eta/core/datasets.py

```diff
@@ -1136,6 +1151,17 @@ def __init__(self, image_path, labels_path):
         super(BuilderImageRecord, self).__init__(image_path, labels_path)
         self.labels_cls = etai.ImageLabels
 
+    def build(self, dir_path, filename, pretty_print=False):
+        data_filename = filename + os.path.splitext(self.data_path)[1]
+        data_path = os.path.join(dir_path, data_filename)
+        etau.copy_file(self.data_path, data_path)
```

Comment:
Will the file be on a local disk path at this time?

Reply:
Yes, via `validated_path`.